src/Analyzers/VisualBasic/Analyzers/IncludeParenthesesForMethodInvocations/VisualBasicIncludeParenthesesForMethodInvocationsDiagnosticAnalyzer.vb

```diff
@@ -0,0 +1,66 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports Microsoft.CodeAnalysis.CodeStyle
+Imports Microsoft.CodeAnalysis.Diagnostics
+Imports Microsoft.CodeAnalysis.Operations
+Imports Microsoft.CodeAnalysis.VisualBasic.CodeStyle
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.IncludeParenthesesForMethodInvocations
+    <DiagnosticAnalyzer(LanguageNames.VisualBasic)>
+    Friend NotInheritable Class VisualBasicIncludeParenthesesForMethodInvocationsDiagnosticAnalyzer
+        Inherits AbstractBuiltInCodeStyleDiagnosticAnalyzer
+
+        Private Shared ReadOnly s_title As New LocalizableResourceString(
+            NameOf(VisualBasicAnalyzersResources.Remove_method_invocation_parentheses), VisualBasicAnalyzersResources.ResourceManager, GetType(VisualBasicAnalyzersResources))
```

Comment:
do you need this tatic?  can this not just be passed into the DIagnosticDescriptor constructor call?